Source/WebKit/UIProcess/glib/WebProcessPoolGLib.cpp

```diff
@@ -119,8 +120,16 @@ void WebProcessPool::platformInitializeWebProcess(const WebProcessProxy& process
 #endif
 
 #if USE(ATSPI)
-    static const char* accessibilityBusAddress = getenv("WEBKIT_A11Y_BUS_ADDRESS");
-    parameters.accessibilityBusAddress = accessibilityBusAddress ? String::fromUTF8(accessibilityBusAddress) : WebCore::PlatformDisplay::sharedDisplay().accessibilityBusAddress();
+    parameters.accessibilityBusAddress = [] {
+        if (auto* address = getenv("WEBKIT_A11Y_BUS_ADDRESS"))
+            return String::fromUTF8(address);
+
+        String& address = sandboxedAccessibilityBusAddress();
+        if (!address.isNull())
+            return address;
```

Comment:
I know this is an existing issue, but I've realized now. Even if it's not common case there might be two contexts, one with the sandbox enabled and the other disabled, so we should always check here if sandbox is enabled in this particular web process pool.

Reply:
Good catch!